lib/src/rules/avoid_field_initializers_in_const_classes.dart

```diff
@@ -99,10 +99,13 @@ class _Visitor extends SimpleAstVisitor<void> {
     if (node.isStatic) return;
     if (!node.fields.isFinal) return;
     // only const class
-    if (node
-        .getAncestor<ClassDeclaration>((e) => e is ClassDeclaration)
-        .declaredElement
-        .constructors
+    ClassDeclaration classDeclaration =
+        node.getAncestor<ClassDeclaration>((e) => e is ClassDeclaration);
+    if (classDeclaration == null) {
```

Comment:
Can we use just `node.parent is ClassDeclaration`?
Fields are always direct children of classes.

Reply:
Done